consensus/propagation/catchup.go

```diff
@@ -121,13 +119,10 @@ func (blockProp *Reactor) AddCommitment(height int64, round int32, psh *types.Pa
 		block:       combinedSet,
 		maxRequests: bits.NewBitArray(int(psh.Total * 2)), // this assumes that the parity parts are the same size
 	}
-	blockProp.Logger.Info("added commitment", "height", height, "round", round)
 
 	// increment the local copies of the height and round
-	blockProp.currentHeight = height + 1
-	blockProp.currentRound = 0
-	blockProp.consensusHeight = height
-	blockProp.consensusRound = 0
+	blockProp.height = height + 1
```

Comment:
keep only the `height` and it refers to the current height we're trying to reach consensus on